src/common/transformations/tests/common_optimizations/nop_elimination.cpp

```diff
@@ -1598,6 +1598,51 @@ TEST_F(TransformationTestsF, EliminateConcatStridedSlice) {
     }
 }
 
+TEST_F(TransformationTestsF, EliminateConcatStridedSliceAll) {
```

Comment:
as I can see, several issues were fixed in this PR, but only one test was implemented
does this test cover all fixed cases? if not, could you please extend test coverage?